src/main/java/uk/gov/hmcts/divorce/common/service/task/SendSubmissionNotifications.java

```diff
@@ -59,8 +59,12 @@ private void sendCitizenNotifications(final CaseData caseData, final Long caseId
             }
         }
 
-        if (caseData.getApplication().hasAwaitingDocuments()) {
-            applicationOutstandingActionNotification.send(caseData, caseId);
+        if (caseData.getApplication().hasAwaitingApplicant1Documents()) {
+            applicationOutstandingActionNotification.sendToApplicant1(caseData, caseId);
```

Comment:
would be nice to add some logging here like app1 has awaiting docs so sending notification ...